.travis.yml

```diff
@@ -13,6 +13,10 @@ matrix:
       env: TOXENV=py35
     - python: 3.6
       env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+      dist: xenial
+      sudo: required
```

Comment:
To be absolutely clear: [this line is now the default behavior](https://changelog.travis-ci.com/linux-builds-run-on-vms-by-default-77106) so it is unnecessary. You can also remove the `sudo: false` at the top of this file.